src/state/CallViewModel.test.ts

```diff
@@ -54,15 +55,23 @@ vi.mock("@livekit/components-core");
 
 const localRtcMember = mockRtcMembership("@carol:example.org", "CCCC");
 const aliceRtcMember = mockRtcMembership("@alice:example.org", "AAAA");
+const aliceDopplegangerRtcMember = mockRtcMembership(
+  "@alice2:example.org",
+  "AAAA",
+);
 const bobRtcMember = mockRtcMembership("@bob:example.org", "BBBB");
 const daveRtcMember = mockRtcMembership("@dave:example.org", "DDDD");
 
-const alice = mockMatrixRoomMember(aliceRtcMember);
+const alice = mockMatrixRoomMember(aliceRtcMember, { rawDisplayName: "Alice" });
+const aliceDoppleganger = mockMatrixRoomMember(aliceDopplegangerRtcMember, {
```

Comment:
```suggestion
const aliceDoppelganger = mockMatrixRoomMember(aliceDopplegangerRtcMember, {
```

Reply:
It makes more sense if you change/refactor this in an ide. All occurances of Doppleganger